resources/resource_transformers/babel/babel.go

```diff
@@ -153,13 +168,21 @@ func (t *babelTransformation) Transform(ctx *resources.ResourceTransformationCtx
 	}
 	cmdArgs = append(cmdArgs, "--filename="+ctx.SourcePath)
 
+	compileOutput, err := ioutil.TempFile("", "compileOut-*.js")
```

Comment:
Can you add a comment to this section as to "why we're doing it" (link to the issue about the stderr issue). I'm going to look at this in 1 year and think "this does not look optimal".

Reply:
Adding comment.